render.py

```diff
@@ -21,10 +21,10 @@
 # VERSION should be a release number or "conference" as in the following examples:
 # VERSION = "13.2"
 # VERSION = "conference"
-VERSION = "42.3"
+VERSION = "15.1"
```

Comment:
You are a bit early ;-) - can we please have a counter for 15.0 first?

Reply:
I agree, and please lets fix this **soon,** because the current counter is still serving 42.3 logos (several websites include the counter URL statically and it would be easy to promote leap 15):

https://countdown.opensuse.org/
https://counter.opensuse.org/